### ISSUE TITLE
clussiana@psi.org on build #10: Why we are not including cases not tested in Suspected…

### ISSUE THREAD

**Issue by ifoche** [EyeSeeTea/SurveillanceLaosApp#13]
Feedback from clussiana@psi.org : Why we are not including cases Not tested in Suspected cases? It seems that suspected cases is the total of +ve and -ve cases, while it should be tested and not tested (answer to the first question).
@rbeccapotter : in case you want to keep Suspected cases as the total of +ve and -ve cases, consider reword this to Tested cases, because this is what it is. In any case, I'd advocate for modifying the math behind this and use Suspected cases to track how many fever cases appear at the facility (Tested + Not tested).

<img src="https://s3-us-west-2.amazonaws.com/buddybuild-screenshots/5774c82ced561801003de3c2/57bc349ec88f110100b8a3fe/1e590637-c5fe-434b-b649-fb457eb030f8.jpg" width="33%" height="33%" /><table><tr><td>Created</td><td>Wed Aug 24 2016 15:54:09 GMT+0000 (UTC)</td></tr><tr><td>Build</td><td>10</td></tr></table>

 [Link to buddybuild feedback from build 10](https://dashboard.buddybuild.com/apps/5774c82ced561801003de3c2/feedback?fid=57bdc321987fc00100b30be7&bnum=10)

**Comment by rbeccapotter** [comment]
@cristinaelle  Yes, Suspected should = Tested+not tested (sum of all Q1).

Actually, this analytics table does not match what we have in the master LA MCS design Google doc. "Tested" (total positive+negative) should be its own separate line in the table. There are several other differences also...

**Comment by ifoche** [comment]
@rbeccapotter, @cristinaelle  you're right, we'll fix it

**Comment by rbeccapotter** [comment]
@ifoche the monitor table  can be updated according to the formulas on Monitor tab here: https://docs.google.com/spreadsheets/d/1703GttDy8uUIRkNfkJqxIJzWBYL5M3bRPd_u7TD6JM0/edit#gid=1277797201
